Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -27,6 +27,10 @@ Write-Verbose "includeFullReport = $includeFullReport"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 
+Write-Warning "We are transferring ownership of the SonarQube related build tasks to Sonar Source. Please replace these build tasks with the ones provided by Sonar Source. 
```

Comment:
Same message: "Ownership of the SonarQube for MSBuild tasks is transfered to SonarSource. Please replace these build tasks by the one brought by SonarSource's extension on the marketplace".